weld-lite-extension-translator/src/main/java/org/jboss/weld/lite/extension/translator/ExtensionInvoker.java

```diff
@@ -30,7 +30,6 @@ class ExtensionInvoker {
         for (Class<? extends BuildCompatibleExtension> extensionClass : extensions) {
             SkipIfPortableExtensionPresent skip = extensionClass.getAnnotation(SkipIfPortableExtensionPresent.class);
             if (skip != null) {
-                // TODO only if the corresponding portable extension exists!
```

Comment:
This TODO is legit. If you don't feel like fixing it, I'd say you should keep it in place :-)

Reply:
Hm, then I probably don't recall the case in which this would be an issue.
How would you be able to reference the class in the annotation without it existing?

Reply:
That's ... a very good point :-) Unless there's a way for a portable extension to be present but not executed, you're right that it's enough.

Reply:
Actually, there is - you can have it on CP but not declared as `META-INF/services` which should cause it not to execute.
But I am not sure it then makes sense to have BCE with this annotation in place :thinking: 
Mainly because containers that cannot run PE will ignore them anyway (and you will never do the skipping) and those that can execute then and use this annotation should run the PE.

Reply:
Yeah, the annotation should probably have been `@SkipInCdiFull` or something like that. The class-valued annotation member servers as a documentation link and that's probably enough...